docs/src/visualization/sharing.rst

```diff
@@ -56,7 +56,7 @@ When your browser connects to auspice.us, it downloads from the server a version
 Option 2: Nextstrain community pages
 ------------------------------------
 
--  **Example:** `CZBiohub's California COVID Tracker <https://nextstrain.org/community/czbiohub/covidtracker/>`__
+-  **Example:** `ESR-NZ's "GenomicsNarrativeSARSCoV2" community builds <https://next.nextstrain.org/community/ESR-NZ/GenomicsNarrativeSARSCoV2>`__
```

Comment:
Let's not use canary URLs (next.nexstrain.org) in docs, but other than this the change looks good